raw_data/preprocessed/africa/zimbabwe_english.txt

```diff
@@ -0,0 +1,486 @@
+
+Cancellation of effluent discharge licence
+(61) The Board may in writing, cancel any effluent discharge licence: [a] if the holder of the licence contravenes any provision of this Act; or [b] if the holder fails to comply with any condition specified in the licence; or [c] if the Board considers it in the interest of the environment or in the public interest so to do.
+
+Register of effluent discharge licences
+(62) [1] The Board shall maintain a register of all effluent discharge licences issued under this Act. [2] The register shall be a public document and may be inspected at any reasonable hour at any premises of the Agency by any person on the payment of the prescribed fee.
+
+Air quality standards
+(63) [1] The Standards and Enforcement Committee shall: [a] advise the Board on how to establish criteria and procedure for the measurement of air quality; [b] recommend to the Board: [i] ambient air quality standards; [ii] occupational air quality standards; [iii] emission standards for various sources; [iv] criteria and guidelines for air pollution control for both mobile and stationary sources; [v] any other air quality standards; [c] advise the Board on measures necessary to reduce existing sources of air pollution by requiring the redesign of plants or the installation of new technology or both, to meet the requirements of standards established under this section; [d] recommend to the Board guidelines to minimise emission of green house gases and identify suitable technologies to minimise air pollution; [e] advise the Board on emissions' concentration and nature of pollutants emitted; [f] recommend to the Board the best practicable technology available in controlling pollutants during the emission process; [g] determine for consideration by the Board the analytical methods for monitoring air contaminants and recommend the establishment of such number of laboratories for analytical services as may be needed; [h] request the Board to carry out investigations of actual or suspected air pollution including pollution produced by aircraft and other self-propelled vehicles and by factories and power generating stations; [i] request the Board to order any industrial or other source of air pollution to file such returns and provide such information as it may require; and [j] do all such things as appear necessary for the monitoring and controlling of air pollution. [2] Any person who emits any substances which cause air pollution (that is, by the emission of substances in excess of a prescribed amount for a particular source) in contravention of emission standards established under this Part shall be guilty of an offence and liable to a fine not exceeding level fourteen or fifteen million dollars, whichever is the greater, or to imprisonment for a period not exceeding five years or to both such fine and such imprisonment. 
+
+[Subsection amended by section 28 of Act 6 of 2005]
+
```

Comment:
Please delete these three lines and append the contents of line 226 to the contents of line 222.